### PR TITLE
AI can now look up and down zlevels

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

AI look up and down verb works now

## Why It's Good For The Game

AI has trouble with multi-z maps right now. the button does nothing

## Changelog
:cl:
add: AI's go up and down commands finally work
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
oh yeah and before if you pressed go up in a multiz map it would fling the AI upwards and then you would immediately fall downwards, that is also fixed by this

**Comment by MMMiracles** [comment]
thank you frog, very cool!

**Comment by Anonmare** [comment]
On Eris, the AI has HUD buttons to move their view up and down, something like that would be useful as a QoL

**Comment by Dennok** [comment]
> On Eris, the AI has HUD buttons to move their view up and down, something like that would be useful as a QoL

I think this buttons need to all. and status of button show you able to move in it dir.

**Comment by actioninja** [comment]
A HUD button for changing z level should be added but that's not what this PR is for.

File: code/modules/mob/living/silicon/ai/freelook/eye.dm [hunk 1]
```diff
@@ -94,6 +94,25 @@
 		if(ai.master_multicam)
 			ai.master_multicam.refresh_view()
 
+//it uses setLoc not forceMove, talks to the sillycone and not the camera mob
+/mob/camera/aiEye/zMove(dir, feedback = FALSE)
+	if(dir != UP && dir != DOWN)
```
Comment (actioninja) [inline code/modules/mob/living/silicon/ai/freelook/eye.dm]: Change this to an or for ultimate micro optimization

Comment (actioninja) [reply code/modules/mob/living/silicon/ai/freelook/eye.dm]: This is wrong but it would be better to do `if(!(dir & UP|DOWN))`

**Pull request merged**